deploy/cic-spec.yaml

```diff
@@ -288,9 +288,9 @@ paths:
                 $ref: "#/components/schemas/OAuthErrorResponse"
       x-amazon-apigateway-request-validator: "both"
       x-amazon-apigateway-integration:
-        httpMethod: "GET"
+        httpMethod: "POST"
```

Comment:
How come this is a POST when the /authorization is a GET?

Reply:
as discussed, this needs to be POST for sending request to the lambda